server/tests/ballot_comparison/test_ballot_comparison_manifests.py

```diff
@@ -0,0 +1,207 @@
+import io
+import json
+from collections import defaultdict
+from flask.testing import FlaskClient
+import pytest
+
+from ...models import *  # pylint: disable=wildcard-import
+from ..helpers import *  # pylint: disable=wildcard-import
+from ...bgcompute import bgcompute_update_ballot_manifest_file
+
+
+# In one jurisdiction, add the Container column to the manifest. In this
+# jurisdiction, ballots should be divvied up between audit boards by container,
+# not the usual tabulator+batch name.
```

Comment:
I assume the tests that already exist test the case where there isn't a container/batch difference?

Reply:
there are existing tests for ballot manifests with no container or tabulator column. this file tests what happens when you include container and tabulator.

Reply:
dope.